deployment/human-connection/service-nginx.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: nitro-web
+  namespace: human-connection
+  labels:
+    human-connection.org/selector: deployment-human-connection-web
+spec:
+  ports:
+    - name: web
+      port: 3000
+      targetPort: 3000
```

Comment:
Shouldn't really matter if you just serve a single html page. You could choose whatever you want, port 80 sounds good to me in case of an ordinary webserver.